src/data/calculate.ts

```diff
@@ -204,9 +204,6 @@ export const MAX_POINTS = 100000
 // in either direction
 export const ERROR_FACTOR = 3
 
-export const parsePopulation = (input: string): number =>
-  Number(input.replace(/[^0-9.e]/g, ''))
-
```

Comment:
Since previously, the website would allow non-numeric characters and simply ignore them, some users may have values like `100,000` in their population field (or even `abc1_0_0_0_0_0`. We should test what will be the impact on such a user.